Sources/OfflineEntitlements/ProductEntitlementMapping.swift

```diff
@@ -16,14 +16,14 @@ import Foundation
 /// A mapping between products and entitlements.
 struct ProductEntitlementMapping {
 
-    var entitlementsByProduct: [String: Set<String>]
+    var entitlementsByProduct: [String: [String]]
 
 }
 
 extension ProductEntitlementMapping {
 
     /// - Returns: entitlement identifiers associated to the given product identifier
-    func entitlements(for productIdentifier: String) -> Set<String> {
+    func entitlements(for productIdentifier: String) -> [String] {
```

Comment:
This was a `Set` but didn't really need to. I changed it because:
1. It's an internal API and can be changed without braking the public API
2. An array is deterministic for unit tests
3. Outside tests, it's only used [here](https://github.com/RevenueCat/purchases-ios/blob/46bfe3cf8640f6e0e22669ce76a16bfb3d683560/Sources/OfflineEntitlements/CustomerInfo%2BOfflineEntitlements.swift#L82-L89) and is still looped over the whole Set, so there's no changed effect when looping through the array  https://github.com/RevenueCat/purchases-ios/blob/46bfe3cf8640f6e0e22669ce76a16bfb3d683560/Sources/OfflineEntitlements/CustomerInfo%2BOfflineEntitlements.swift#L82-L89
4. With an array we can deterministically add Snapshot tests for product entitlement mapping responses